docs/zq2/index.md

```diff
@@ -4,16 +4,53 @@ title: Zilliqa 2
 keywords: []
 ---
 
----
+# Zilliqa 2: An Overview
 
-Zilliqa 2 is the next generation of the Zilliqa blockchain.
-It is currently in active development at https://github.com/Zilliqa/zq2.
-It is not yet stable or complete, feel free to create an issue in GitHub when you find problems.
+## Introduction
 
-For more information please refer to the [FAQ](faq.md).
+Zilliqa 2 is the next generation of the Zilliqa blockchain. It represents a fundamental overhaul of Zilliqa's blockchain node, introducing substantial improvements in network performance and utility while preserving all the features of the current Zilliqa version, including full EVM (Ethereum Virtual Machine) compatibility.
+
+It is currently under development, so not yet stable or complete, feel free to create an issue in [GitHub](https://github.com/Zilliqa/zq2) when you find problems.
+
+## Key Enhancements in Zilliqa 2
+
+Zilliqa 2 brings forth several enhancements to elevate its performance and scalability:
+
+- **Faster Finality**: Introducing a new consensus algorithm (Fast-HotStuff) for quicker transaction confirmations.
+
+- **Improved Scalability and Sharding**: The upgrade empowers developers with an enhanced sharding system, providing the flexibility to create custom shards with programmable security parameters.
+
+- **Developer-Focused Design**: Prioritizing an enhanced developer experience, Zilliqa 2.0 aims to simplify blockchain development and streamline user transactions.
+
+- **Programmable Model and EVM Compatibility**: The upgrade embraces a programmer's model heavily based on EVM, allowing smart contracts to modify blockchain behavior without altering underlying code. Full support for both Scilla and EVM is retained.
```

Comment:
I think its a bit vague and unclear what exactly this means. The point about EVM + Scilla compatibility is good, but I'm less sure about "programmable model" / "programmer's model based on EVM"?

Reply:
Will remove the reference to "programmable model"